config/routes.rb

```diff
@@ -9,6 +9,10 @@
   # You can have the root of your site routed with "root"
   root 'welcome#index'
 
+  resources :list do
```

Comment:
I don't mean omit the entire thing, I mean just doing `resources :list` instead of `resources :list do end`.